### PR TITLE
Capistrano based deployment

### PR DESCRIPTION
**Pull request by rrreeeyyy**
Hi, we can deploy with capistrano for now!
So this is initial PR for automated deployment.

I'll send a another PR that updates wercker configuration, after this PR merged.

Could you review it?

### CODE REVIEW

File: Capfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+#   https://github.com/capistrano/rvm
+#   https://github.com/capistrano/rbenv
+#   https://github.com/capistrano/chruby
+#   https://github.com/capistrano/bundler
+#   https://github.com/capistrano/rails
+#   https://github.com/capistrano/passenger
+#
+# require "capistrano/rvm"
+require "capistrano/rbenv"
```
Comment (sawa-zen) [inline Capfile]: I think that nodebrew should specify the version of node.js. What do you think?

File: config/deploy/production.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+server "blog.topotal.com", roles: %w(app migration)
```
Comment (sawa-zen) [inline config/deploy/production.rb]:
```
set :deploy_to, "/var/www/blog.topotal.com"
```
Is not this necessary?

File: Capfile (hunk 1 above)
Comment (rrreeeyyy) [inline Capfile]: Exactly, we should use capistrano-nodebrew if node command installed by nodebrew.
But, u know, currently node command installed by yum command.

File: config/deploy/production.rb (hunk 2 above)
Comment (rrreeeyyy) [inline config/deploy/production.rb]: Because it is inherited by `config/deploy.rb`

File: Capfile (hunk 1 above)
Comment (ics-sawada) [inline Capfile]: Oh... There is no problem if it is. :ok_man:

File: config/deploy/production.rb (hunk 2 above)
Comment (ics-sawada) [inline config/deploy/production.rb]: I see 🙆

**Comment by hbadmin** [comment]
👍

**Comment by nari-ex** [comment]
:bow:

**Pull request merged**